jekyll/_ccie/overview.md

```diff
@@ -50,12 +69,17 @@ Our Builder Boxes handle running your builds, and store no state themselves. Eac
 | Services Box                     | all traffic / all ports | Internal Communication                                         |
 | Builder Boxes (including itself) | all traffic / all ports | Internal Communication                                         |
 
-#### GitHub
----
+CircleCI Enterprise uses GitHub Enterprise credentials for
+authentication which, in turn, may use LDAP, SAML, or SSH for access. The following table describes the ports used on machines running GitHub to communicate with the Services and Builder instances.
 
 | Source        | Ports   | Use          |
 |---------------|---------|--------------|
-| Services Box  | 22      | Git Access   |
-| Services Box  | 80, 443 | API Access   |
-| Builder Boxes | 22      | Git Access   |
-| Builder Boxes | 80, 443 | API Access   |
+| Services   | 22      | Git Access   |
+| Services   | 80, 443 | API Access   |
+| Builder  | 22      | Git Access   |
+| Builder  | 80, 443 | API Access   |
+
```

Comment:
We should probably add the high port ranges we use for SSH to the containers, as many use that feature.